test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -79,24 +79,32 @@ elif [[ $FRAMEWORK == "mxnet" ]]; then
         echo "SMDebug tests failed for $FRAMEWORK"
         exit $exit_code
     fi
-else
-    TENSORFLOW_DATASETS_VERSION="4.0.1"
-    if [[ $FRAMEWORK == "tensorflow2" ]]; then
-        tf_version=$(python -c "import tensorflow as tf; print(tf.__version__)")
-        if [[ "$tf_version" == *"2.0."* ]]; then
-            TENSORFLOW_DATASETS_VERSION="3.2.1"
-        fi
-    fi
-    # pip install tensorflow-datasets==${TENSORFLOW_DATASETS_VERSION} "absl-py>=0.9,<0.11"
+
+elif [[ $FRAMEWORK == "tensorflow2" ]]; then
+    tf_version=$(python -c "import tensorflow as tf; print(tf.__version__)")
     for test_file in tests/${FRAMEWORK}/*
     do
-        
-        if [[ "$tf_version" == *"2.7."* && "$test_file" == *"test_keras_mirrored.py" ]]; then
-            echo "skipping flaky test_keras_mirrored.py"
+      if [[ "$tf_version" == *"2.7."* && "$test_file" == *"test_keras_mirrored.py" ]]; then
```

Comment:
nit: indentation looks inconsistent. Should be 4 spaces?

Reply:
Please have someone from `smdebug` team review the changes to this file.

Reply:
@MZSHAN had provided a review. I will double check regarding this file. Thanks.

Reply:
Needs to be skipped for 2.8 as well.